tests/unittests/Foundation/NSPointerArrayTests.mm

```diff
@@ -61,9 +55,12 @@ @implementation NSPointerArraySubClass
     ASSERT_EQ_MSG(array.count, 3, @"should have 3 items in the array.");
 }
 
-TEST(NSPointerArray, Subclassing) {
+// Note: On the reference platform when a class that does not support sub-classing is sub-classed, the parent class is
```

Comment:
This feels more like a WindowsOnly test, going by the standards we laid out this morning.